src/test/rustdoc-ui/doc-alias-assoc-const.rs

```diff
@@ -0,0 +1,48 @@
+#![feature(doc_alias)]
+#![feature(trait_alias)]
+
+pub struct Foo;
+
+pub trait Bar {
+    const BAZ: u8;
+}
+
+impl Bar {
+    #[doc(alias = "CONST_BAZ2")] //~ ERROR
+    const BAZ: u8 = 0;
+}
+
+mod foo {
+    pub trait Fuck { const X: u32; }
```

Comment:
... maybe we should pick another name for the trait.

Reply:
Oh damn, forgot to rename before pushing. ><